src/meshes/mesh_io.jl

```diff
@@ -222,9 +222,13 @@ function save_mesh_file(mesh::P4estMesh, output_directory, timestep, mpi_paralle
 end
 
 # TODO: Implement this function as soon as there is support for this in `t8code`.
-function save_mesh_file(mesh::T8codeMesh, output_directory, timestep, mpi_parallel)
-    error("Mesh file output not supported yet for `T8codeMesh`.")
+# function save_mesh_file(mesh::T8codeMesh, output_directory, timestep, mpi_parallel)
+#     error("Mesh file output not supported yet for `T8codeMesh`.")
+
+#     return joinpath(output_directory, "dummy_mesh.h5")
+# end
 
+function save_mesh_file(mesh::T8codeMesh, output_directory, timestep, mpi_parallel)
     return joinpath(output_directory, "dummy_mesh.h5")
 end
```

Comment:
All those plotting routines are very improvised and are not in the standard Trixi way.
I use the t8code internal routines to directly create `vtu` files inside Trixi.jl.
They probably should be deleted or changed for this PR.